crates/engine/tree/src/persistence.rs

```diff
@@ -135,39 +135,31 @@ impl<DB: Database> PersistenceService<DB> {
         Ok(())
     }
 
-    // TODO: perform read for td and start
     /// Writes the transactions to static files, to act as a log.
     ///
     /// The [`update_transaction_meta`](Self::update_transaction_meta) method should be called
     /// after this, to update the checkpoints for headers and block bodies.
-    fn log_transactions(
-        &self,
-        block: Arc<SealedBlock>,
-        start_tx_number: u64,
-        td: U256,
-    ) -> ProviderResult<u64> {
-        debug!(target: "tree::persistence", ?td, ?start_tx_number, "Logging transactions");
+    fn log_transactions(&self, block: Arc<SealedBlock>) -> ProviderResult<u64> {
```

Comment:
I find `log_` very confusing can we stick to `write` ?

Reply:
sure, renamed